src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/EntityPublished/changeRequest.html.twig

```diff
@@ -0,0 +1,42 @@
+{% extends '::base.html.twig' %}
+
+{% block body_container %}
+    <h1>{% block page_heading %}{{ 'entity.change_request.title'|trans }}{%endblock%}</h1>
+
+    <div class="fieldset card action">
+        {% include '@Dashboard/EntityActions/actionsForDetail.html.twig' with {entity: actions, isAdmin: isAdmin} %}
+    </div>
+
+    <div class="fieldset card">
+        {% if changeRequests.getChangeRequests()|length %}
+            <div class="wysiwyg">{{ 'entity.change_request.text.html'|trans|wysiwyg }}</div>
+        {% else %}
+            <div class="wysiwyg">{{ 'entity.change_request.none.text.html'|trans|wysiwyg }}</div>
+        {% endif %}
+    </div>
+
+    {%  for changeRequest in changeRequests.getChangeRequests() %}
+
+        <div class="fieldset card">
+            <h2>{{ changeRequest.created|date }}</h2>
+
+            {% if changeRequest.note is not null %}
+                <p>{{ changeRequest.note }}</p>
+            {% endif %}
+
+            <table>
+                <tr>
+                    <th>{{ 'entity.change_request.value'|trans }}</th>
```

Comment:
Setting the width on the first column makes more sense to me. At this point the attribute name is much wider than the possibly longer value it can have.

Also your attempt at naming the `th` class is quite good. But while the rest of the block is not named accordingly, maybe it is better to just name it in a more conventional manner. For example set the classname on the table. And target the first th from the stylesheet.

```
table.change-request-overview th:first-child {
    width: 24em;
    overflow: hidden;
}
```